src/com/jambit/Database.java

```diff
@@ -0,0 +1,48 @@
+package com.jambit;
+
+import java.sql.*;
+
+public class Database {
+  private static final String HOST = "azubipi1.client.jambit.com";
+  private static final String USERNAME = "root";
+  private static final String PASSWORD = "123";
+  private static final String DATABASE = "new_schema";
+  private static final int PORT = 3306;
+
+  private Connection con;
+
+  public Database() {
+    connect();
+  }
+
+  private void connect() {
+    try {
+      con =
+          DriverManager.getConnection(
+              "jdbc:mysql://" + HOST + ":" + PORT + "/" + DATABASE, USERNAME, PASSWORD);
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
+  }
+
+  public void basicRead(String table) {
```

Comment:
1. this method does not return anything...
2. use specific methods for retrieving the data you like to fetch. e.g. 
public ArrayList<MoodEntry> fetchAllMoodEntries()
public ArrayList<MoodEntry> fetchMoodEntriesByTime(Date usefuelName)
...